Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -4550,6 +4550,8 @@ def list_vulnerabilities_by_software_command(client: MsClient, args: dict) -> li
     vulnerabilities_response_value = vulnerabilities_response.get('value')
     demisto.debug(vulnerabilities_response_value)
```

Comment:
demisto.debug(f'Vulnerabilities Response {vulnerabilities_response_value}, with length of {len(vulnerabilities_response_value)}')